src/utils/erc7201.ts

```diff
@@ -0,0 +1,9 @@
+import { keccak256 } from 'ethereum-cryptography/keccak.js';
+import { utf8ToBytes, hexToBytes, bytesToHex } from 'ethereum-cryptography/utils.js';
+
+export function erc7201Location(id: string): string {
+  const a = keccak256(utf8ToBytes(id));
+  const b = BigInt('0x' + bytesToHex(a)) - 1n;
+  const c = hexToBytes(b.toString(16).padStart(64, '0'));
+  return '0x' + bytesToHex(keccak256(c));
+}
```

Comment:
If we have ethers v6, we could do it as
```suggestion
import { ethers } from 'ethers';

export function erc7201Location(id: string): string {
    return ethers.keccak256(ethers.zeroPadValue(
        ethers.toBeArray(ethers.toBigInt(ethers.keccak256(ethers.toUtf8Bytes(id))) - 1n), 
        32
    ));
}
```